libecho/src/echo.cpp

```diff
@@ -1,17 +1,17 @@
 #include "echo.h"
-#include "AudioOutput.h"
-#include "AudioInput.h"
 #include "AudioFormatFactory.h"
+#include "AudioInput.h"
+#include "AudioOutput.h"
 #include "QTInitializer.h"
 
-void echo::send(const std::vector<uint8_t>& buffer) {
+void echo::send(const std::vector<uint8_t> &buffer) {
     static AudioOutput audio(AudioFormatFactory::getDefaultOutputFormat());
 
     /* Example transmission */
-    const size_t atOnce = 260; // (bitrate / notify_interval) + eps;
+    const size_t atOnce = 260;  // (bitrate / notify_interval) + eps;
     audio.enqueueData(reinterpret_cast<const char *>(buffer.data()), std::min(buffer.size(), atOnce));
 
-    for(int i = atOnce; i <  buffer.size(); i += atOnce) {
+    for (int i = atOnce; i < buffer.size(); i += atOnce) {
```

Comment:
sugerowałbym int –> size_t aby wyciszyć warninga i zachować spójność używanych typów

Reply:
Tak jak wyżej - mogę zmienić ale to trochę bezcelowe